app/serializers/user_game_serializer.rb

```diff
@@ -8,4 +8,12 @@ class UserGameSerializer
   attribute :game_name do |object|
     object.game&.name
   end
+
+  attribute :player_count do |object|
+    object.game&.users.count
+  end
+
+  attribute :rounds do |object|
+    "#{object.game&.rounds.count}/#{object.game&.length_in_days}"
```

Comment:
Look good, only future thing we need to take into account, is possibly how we want to handle games that have come to completion with all rounds.